worlds/sc2/Options.py

```diff
@@ -761,8 +761,8 @@ class StartingSupplyPerItem(Range):
     """
     display_name = "Starting Supply Per Item"
     range_start = 0
-    range_end = 200
-    default = 5
+    range_end = 8
```

Comment:
I'd set it to 16 (2 supply depots as you get from max minerals)

IIRC with some asyncs there were YAMLs with setting to 10